LINK/usr/bin/evmexport

```diff
@@ -0,0 +1,133 @@
+#!/usr/bin/env bash
+
+assert_num_args () {
+  num_args_received=$1
+  num_args_expected=$2
+  if [ "$num_args_received" -ne "$num_args_expected" ]
+  then
+    echo "Error: Wrong number of arguments (expected=$num_args_expected received=$num_args_received)"
+    usage
+    exit 1
+  fi
+}
+
+assert_export_dir_exists () {
+  export_dir=$1
+  if [ ! -d "$export_dir" ]
+  then
+    echo "Export directory doesn't exist! [${export_dir}]"
+    exit 1
+  fi
+}
+
+op_alerts () {
+  assert_num_args $# 1
+  EXPORT_DIR=$(readlink -v -f "$1")
+  assert_export_dir_exists "$EXPORT_DIR"
+
+  pushd /var/www/miq/vmdb > /dev/null
+  bin/rake "evm:export:alerts[${EXPORT_DIR},keep_spaces=${KEEP_SPACES}]"
+  popd > /dev/null
+}
+
+op_alertprofiles () {
+  assert_num_args $# 1
+  EXPORT_DIR=$(readlink -v -f "$1")
+  assert_export_dir_exists "$EXPORT_DIR"
+
+  pushd /var/www/miq/vmdb > /dev/null
+  bin/rake "evm:export:alertsets[${EXPORT_DIR},keep_spaces=${KEEP_SPACES}]"
+  popd > /dev/null
+}
+
+contains () {
+  SEARCH_TERM=$1
+  shift
+  LIST=$@
+
+  for element in $LIST; do
+    if [[ "$SEARCH_TERM" == "$element" ]]; then
+      return 0
+    fi
+  done
+
+  return 1
+}
+
+parse_arguments () {
```

Comment:
I think I would rather not have a function for this.

1. I usually expect option parsing to be in the main script and
2. I know it works, but it feels odd to be setting `remaining_args` in a function and then using it after.